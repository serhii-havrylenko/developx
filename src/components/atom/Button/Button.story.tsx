// @flow
import { action } from '@storybook/addon-actions';
import { storiesOf } from '@storybook/react';
import * as React from 'react';

import Button from '#atom/Button';

const stories = storiesOf('StepItem', module);

stories.add('simple btn', () => <Button onClick={action('clicked')} />);
