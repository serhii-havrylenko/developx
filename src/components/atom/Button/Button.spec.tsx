import { shallow } from 'enzyme';
import * as React from 'react';

import Button from './Button';

describe('<Button /> tests', () => {
  test('should match snapshot', () => {
    const tree = shallow(<Button onClick={undefined} />);
    expect(tree.dive()).toMatchSnapshot();
  });
});
