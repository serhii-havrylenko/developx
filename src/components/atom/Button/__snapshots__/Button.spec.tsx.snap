// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> tests should match snapshot 1`] = `
<Button
  classes={
    Object {
      "colorInherit": "MuiButton-colorInherit-22",
      "contained": "MuiButton-contained-12",
      "containedPrimary": "MuiButton-containedPrimary-13",
      "containedSecondary": "MuiButton-containedSecondary-14",
      "disabled": "MuiButton-disabled-21",
      "extendedFab": "MuiButton-extendedFab-19",
      "fab": "MuiButton-fab-18",
      "flat": "MuiButton-flat-6",
      "flatPrimary": "MuiButton-flatPrimary-7",
      "flatSecondary": "MuiButton-flatSecondary-8",
      "focusVisible": "MuiButton-focusVisible-20",
      "fullWidth": "MuiButton-fullWidth-26",
      "label": "MuiButton-label-2",
      "mini": "MuiButton-mini-23",
      "outlined": "MuiButton-outlined-9",
      "outlinedPrimary": "MuiButton-outlinedPrimary-10",
      "outlinedSecondary": "MuiButton-outlinedSecondary-11",
      "raised": "MuiButton-raised-15",
      "raisedPrimary": "MuiButton-raisedPrimary-16",
      "raisedSecondary": "MuiButton-raisedSecondary-17",
      "root": "MuiButton-root-1",
      "sizeLarge": "MuiButton-sizeLarge-25",
      "sizeSmall": "MuiButton-sizeSmall-24",
      "text": "MuiButton-text-3",
      "textPrimary": "MuiButton-textPrimary-4",
      "textSecondary": "MuiButton-textSecondary-5",
    }
  }
  color="primary"
  component="button"
  disableFocusRipple={false}
  disabled={false}
  fullWidth={false}
  mini={false}
  size="medium"
  type="button"
  variant="text"
>
  sample
</Button>
`;
